### ISSUE TITLE
The information about the tracker seems to be quite outdated on marvelsnap.pro and marvelsnapzone

### ISSUE THREAD

**Issue by LMorn** [Razviar/marvelsnaptracker#49]
It still says there's no overlay, being the main thing. And I assume you obtain some data from the game's running memory now, because the overlay sees things way before the player can see them (like it knows if I won or lost as soon as the last turn starts to play out, or that a card was revealed before it flips, or that the latest area revealed created squirrels)

**Comment by androiski** [comment]
Yeah the website still says stuff like no Overlay and that trackers and such aren't legal. But we know that the tracker from Untapped does just the same.
